app/javascript/components/enrollment/steps/ExposureInformation.js

```diff
@@ -27,11 +26,12 @@ class Exposure extends React.Component {
       originalAssignedUser: this.props.currentState.patient.assigned_user,
       assigned_users: this.props.assigned_users,
       selected_jurisdiction: this.props.selected_jurisdiction,
+      isEditMode: window.location.href.includes('edit'),
```

Comment:
same here

Reply:
fixed in https://github.com/SaraAlert/SaraAlert/pull/975/commits/441588e6694ae61c4bd1da5b7da3be4de8fd5aa9